server/requirements.txt

```diff
@@ -73,3 +73,5 @@ win32-setctime==1.1.0 ; python_version >= "3.9" and python_version < "3.13" and
 wrapt==1.15.0 ; python_version >= "3.9" and python_version < "3.13"
 xxhash==3.3.0 ; python_version >= "3.9" and python_version < "3.13"
 yarl==1.9.2 ; python_version >= "3.9" and python_version < "3.13"
+# Custom 4-bit GEMM AWQ kernels
+git+https://github.com/mit-han-lab/llm-awq.git@f084f40bd996f3cf3a0633c1ad7d9d476c318aaa#subdirectory=awq/kernels
```

Comment:
Is there anyway we could move to versioned releases ? 
Works currently it's a sha on main which is good enough for stability imo.

Reply:
@Narsil: Not sure about licensing issues, but can we copy over the kernels to TGI repo from AWQ repo? That's all this `pip install` is pulling in. I saw somewhere that you have a `custom_kernels` directory in the repo.

Reply:
Would recommend switching to `autoawq==0.1.0` as this has support for Windows as well. The `mit-han-lab` kernels break on Windows without the modifications I made.

Reply:
Thanks @casper-hansen.

@Narsil: Okay to have a dependency on `autoawq==0.1.0`?